docs/book/src/roadmap.md

```diff
@@ -14,18 +14,27 @@ This roadmap is a constant work in progress, subject to frequent revision. Dates
 - [Machine failure remediation](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20191030-machine-health-checking.md)
 - [Provider end to end testing framework](https://github.com/kubernetes-sigs/cluster-api/blob/bf51a2502f9007b531f6a9a2c1a4eae1586fb8ca/docs/proposals/20191016-e2e-test-framework.md)
 
-## v0.4 (v1alpha4) ~ July 2020
-
-- [Machine bootstrap failure detection](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/972)
+## v0.3.x (v1alpha3+) ~ June/July 2020
+
+- [E2E Test plan](https://docs.google.com/spreadsheets/d/1uB3DyacOLctRjbI6ov7mVoRb6PnM4ktTABxBygt5sKI/edit#gid=0)
+- [Enable webhooks in integration tests](https://github.com/kubernetes-sigs/cluster-api/issues/2788)
+- [KCP robustness](https://github.com/kubernetes-sigs/cluster-api/issues/2753)
+- [KCP adoption](https://github.com/kubernetes-sigs/cluster-api/issues/2214)
+- [Clusterctl templating](https://github.com/kubernetes-sigs/cluster-api/issues/2339)
+- [Clusterctl cert-manager lifecycle](https://github.com/kubernetes-sigs/cluster-api/issues/2635)
+- [Post apply YAML experiment](https://github.com/kubernetes-sigs/cluster-api/issues/2395)
+- [MHC Support for control-plane nodes](https://github.com/kubernetes-sigs/cluster-api/issues/2836)
+- [Remote workload cluster watcher](https://github.com/kubernetes-sigs/cluster-api/issues/2414)
 - [Improved status conditions](https://github.com/kubernetes-sigs/cluster-api/issues/1658)
+- [Spot instances support](https://github.com/kubernetes-sigs/cluster-api/issues/1876)
+- [Machine bootstrap failure detection](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/972)
 - [Machine load balancers](https://github.com/kubernetes-sigs/cluster-api/issues/1250)
 - [Extensible machine pre-delete hooks](https://github.com/kubernetes-sigs/cluster-api/issues/1514)
-- Define clusterctl inventory specification & have providers implement it
-- Figure out how to change configurations through rolling updates (e.g. apiserver flags, DNS/addons, etc.)
-- [Spot instances support](https://github.com/kubernetes-sigs/cluster-api/issues/1876)
 
-## v0.5 (v1alpha5? v1beta1?) ~ November 2020
+## v0.4 (v1alpha4) ~ Q4 2020
 
+- Define clusterctl inventory specification & have providers implement it
+- Figure out how to change configurations through rolling updates (e.g. apiserver flags, DNS/addons, etc.)
```

Comment:
Should we add https://github.com/kubernetes-sigs/cluster-api/issues/2318 to v0.4?